css-brochure-form/styles.css

```diff
@@ -0,0 +1,78 @@
+div.input {
+  text-align: center;
+}
+
+div.label {
+  padding-left: 653px;
+}
```

Comment:
As we touched on this morning, we want to avoid such large padding values. It usually means we're doing something hacky to get something to work